BootloaderCommonPkg/Include/Library/ConfigDataLib.h

```diff
@@ -74,6 +74,35 @@ typedef struct {
   UINT32  TotalLength;
 } CDATA_BLOB;
 
+//
+// This data structure is same as CDATA_BLOB
+// Its represents contents stored to flash.
+//
+typedef struct {
+  UINT32  Signature;
+  //
+  // This header Length
+  //
+  UINT8   HeaderLength;
+  UINT8   Attribute;
+  //
+  // Security version number
+  // This is available only in flash. It would be overwritten by CDATA_BLOB.InternalDataOffset in runtime
+  //
+  UINT8   Svn;

```

Comment:
Please reuse structure CDATA_BLOB
Define a "union" for  "InternalOffset" and "Svn"

Reply:
Done. Defined union.